softdevice_controller/CHANGELOG.rst

```diff
@@ -60,7 +60,7 @@ Known issues and limitations
 * Synthesized low frequency clock source is not tested or intended for use with the Bluetooth LE stack (DRGN-6362).
 * If an extended advertiser is configured with limited duration, it will time out after the first primary channel packet in the last advertising event (DRGN-10367).
 * The HCI Receiver and Transmitter Test commands are not available.
-  To perform a radio test, utilize the DTM application in the nRF5 SDK (DRGN-12259).
+  To perform a radio test, use :ref:`nrf:_direct_test_mode` or the DTM application in the nRF5 SDK (DRGN-12259).
```

Comment:
It should say nRF5 SDK for nrf52 and the _direct_test_mode link for nrf53.
AFAIK the direct_test_mode sample has only been tested on nrf53 for now.